modules/ROOT/pages/database-administration/standard-databases/create-databases.adoc

```diff
@@ -214,6 +216,57 @@ The `IF NOT EXISTS` and `OR REPLACE` parts of these commands cannot be used toge
 ====
 
 
+==== Create databases using seeding options
+
+Neo4j has built-in support for a seed from a mounted file system (file), FTP server, HTTP/HTTPS server, Amazon S3, Google Cloud Storage, and Azure Cloud Storage.
+The seed provider is determined by the xref:configuration/configuration-settings.adoc#config_dbms.databases.seed_from_uri_providers[dbms.databases.seed_from_uri_providers] setting.
+By default, Neo4j enables support for Amazon S3, Google Cloud Storage, and Azure Cloud Storage.
+To use other providers, you must configure `dbms.databases.seed_from_uri_providers` accordingly.
+
+.Seed providers supported in Neo4j
+[cols="2,1,3",options="header"]
+|===
+| Seed provider
+| URL scheme
+| URI example
+
+| `FileSeedProvider`
+| `file:`
+| `file:/tmp/backup1.backup`
+
+| `URLConnectionSeedProvider`footnote:[From 2025.01, `URLConnectionSeedProvider` does not support `file`.]
+| `ftp:` +
+`http:` +
+`https:`
+| `ftp://myftp.com/backups/backup1.backup` +
+`/http://myhttp.com/backups/backup1.backup` +
+`/https://myhttps.com/backups/backup1.backup`
+
+| `S3SeedProvider` label:deprecated[Deprecated in 5.26]
+| `s3:`
+| `s3://mybucket/backups/backup1.backup`
+
+| `CloudSeedProvider`
```

Comment:
```suggestion
| `CloudSeedProvider` label:default[Default]
```

Reply:
This is default value of the config starting from 2025.01 only. I don't think we need such details here.